src/controllers/accountController.js

```diff
@@ -3,6 +3,11 @@ import moment from 'moment';
 
 import db from '../models/index';
 
+const text = `INSERT INTO
```

Comment:
Any reason why this is outside the class

Reply:
I reused it so I wanted it to be outside, I could also have put it inside a constructor for access that's another thought. Code metrics was showing my code was becoming too complex so I was thinking of making it simpler. Should I put it inside a constructor? I don't know since I'm using static methods